file_snapshot.go

```diff
@@ -384,6 +384,11 @@ func (s *FileSnapshotSink) Close() error {
 	// Close the open handles
 	if err := s.finalize(); err != nil {
 		s.logger.Printf("[ERR] snapshot: Failed to finalize snapshot: %v", err)
+		s.logger.Printf("[INFO] snapshot: Deleting temporary files...")
```

Comment:
I'd get rid of this `INFO` line since we have an error if it fails, and it's pretty expected that it will clean up.